plugins/plugin-template/webpack.config.js

```diff
@@ -28,7 +29,7 @@ module.exports = {
     ],
   },
   resolve: {
-    extensions: [".ts", ".js", ".json"],
```

Comment:
Why do we toke out the .json? if we need that json in the build

Reply:
I think that we better leave it because most of the developers don't know to handle the webpack.config and its not that rare that you need a JSON file in your code

Reply:
fixed